Modules/IO/JPEG/include/itkJPEGImageIO.h

```diff
@@ -117,7 +117,7 @@ class ITKIOJPEG_EXPORT JPEGImageIO : public ImageIOBase
   PrintSelf(std::ostream & os, Indent indent) const override;
 
   void
-  WriteSlice(std::string & fileName, const void * const buffer);
+  WriteSlice(const std::string & fileName, const void * const buffer);
```

Comment:
This really looks like a "constness bug" that should be fixed anyway. Right? (Independent of using std::filesystem.) Maybe even for ITK 5.4?

Reply:
I second the motion to make this a separate PR which can be merged immediately.

Reply:
done, and done.